stdlib/public/Platform/msvcrt.swift

```diff
@@ -13,3 +13,16 @@
 @_exported import ucrt // Clang module
 @_exported import visualc // Clang module
 
+// Unfortunately, the clang importer does not import "complex" macros such as
+// the one that are used in the UCRT for defining `stdin`, `stdout`, `stderr`.
+// Replicate them manually.
+public var stdin: UnsafeMutablePointer<FILE> { return __acrt_iob_func(0) }
```

Comment:
These variables are already define in `Platform.swift` under a guard. Wouldn't it be better to have them in the same place?

Reply:
*sigh* I had forgotten about that.  Thanks!  Yes, these belong there.